libraries/core_libs/network/src/tarcap/packets_handlers/pbft_sync_packet_handler.cpp

```diff
@@ -133,7 +143,7 @@ void PbftSyncPacketHandler::process(const PacketData &packet_data, const std::sh
   LOG(log_tr_) << "Synced PBFT block hash " << pbft_blk_hash << " with " << sync_block.cert_votes.size()
                << " cert votes";
   LOG(log_tr_) << "Synced PBFT block " << sync_block;
-  pbft_mgr_->syncBlockQueuePush(std::move(sync_block), packet_data.from_node_id_);
+  pbft_mgr_->syncBlockQueuePush(std::move(sync_block), packet_data.from_node_id_, cert_votes);
```

Comment:
what happens if cert_votes is empty due to pbft_chain_synced == false ?

Reply:
From Miho branch, I have move the question to https://github.com/Taraxa-project/taraxa-node/pull/1744#discussion_r891665937